Dockerfile

```diff
@@ -1,10 +1,14 @@
 FROM python:3.6-slim
 
+# Make the /app dir
+RUN mkdir /app
+
 # Set the working directory to /app
 WORKDIR /app
 
-# Copy the current directory contents into the container at /app
-COPY . /app
+# Copy required files to /app
+COPY requirements.txt .
```

Comment:
In order to save layers, wouldn't it make sense to copy them all at once using
`COPY requirements.txt fritzinfluxdb.py .`

Reply:
good point, thanks. Fixed with latest push